tests/check/check_signed_video_sign.c

```diff
@@ -498,6 +498,33 @@ START_TEST(correct_signed_stream_with_eos)
 END_TEST
 #endif
 
+START_TEST(check_the_priv_key)
```

Comment:
It is a great idea to have this kind of test since it verifies that we can reconstruct the private key and set it in ONVIF Media Signing. But, the placement of the test in the code is off. Please, put it in the same order as it is called, that is, as a last test.